source/intro.tex

```diff
@@ -424,8 +424,8 @@
 
 \pnum
 Throughout this International Standard, each example is introduced by
-``\enterexample'' and terminated by ``\exitexample''. Each note is
-introduced by ``\enternote'' and terminated by ``\exitnote''. Examples
+``\begin{example}'' and terminated by ``\end{example}''. Each note is
```

Comment:
@tkoeppe I agree. Would you like me to adjust my branch? Is there anything else I can do at this time to assist with this PR?